src/airflow/dags/meta_data.py

```diff
@@ -36,7 +36,7 @@ def load(data: pd.DataFrame):
         from airqo_etl_utils.constants import JobAction
 
         big_query_api = BigQueryApi()
-        big_query_api.save_data(
+        big_query_api.load_data(
             dataframe=data,
             table=big_query_api.sites_table,
             job_action=JobAction.OVERWRITE,
```

Comment:
Thanks @NoahNsimbe  ...interested to continue this discussion about overwrite/update of historical data.
I am starting to get uncomfortable with it.
I sense a future where we do not have to update or overwrite existing time series data.......happy to hear suggestions on this.